src/webapi/date.rs

```diff
@@ -1,22 +1,903 @@
+            1 => js!( return @{self.as_ref()}.setUTCHours(@{args[0]}); ).try_into().unwrap(),
+            _ => js!( return @{self.as_ref()}.setUTCHours(); ).try_into().unwrap()
+        }
+    }
+
+    ///
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCMilliseconds)
+    #[inline]
+    fn set_utc_milliseconds( &self, _milliseconds_value: f64 ) -> f64 {
+        js!( return @{self.as_ref()}.setUTCMilliseconds(@{_milliseconds_value}); ).try_into().unwrap()
+    }
+
+    ///
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCMinutes)
+    #[inline]
+    fn set_utc_minutes( &self, args: &[i32] ) -> f64 {
+        // args represents:
+        // [<minutes_value>, <seconds_value>, <ms_value>]
+        match args.len() {
+            3 => js!(
+                return @{self.as_ref()}.setUTCMinutes(@{args[0]}, @{args[1]}, @{args[2]});
+            ).try_into().unwrap(),
+            2 => js!( return @{self.as_ref()}.setUTCMinutes(@{args[0]}, @{args[1]}); ).try_into().unwrap(),
+            1 => js!( return @{self.as_ref()}.setUTCMinutes(@{args[0]}); ).try_into().unwrap(),
+            _ => js!( return @{self.as_ref()}.setUTCMinutes(); ).try_into().unwrap()
+
+        }
+    }
+
+    ///
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCMonth)
+    #[inline]
+    fn set_utc_month( &self, args: &[i32] ) -> f64 {
+        // args represents:
+        // [<month_value>, <day_value>]
+        match args.len() {
+            2 => js!( return @{self.as_ref()}.setUTCMonth(@{args[0]}, @{args[1]}); ).try_into().unwrap(),
+            1 => js!( return @{self.as_ref()}.setUTCMonth(@{args[0]}); ).try_into().unwrap(),
+            _ => js!( return @{self.as_ref()}.setUTCMonth(); ).try_into().unwrap()
+
+        }
+    }
+
+    ///
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setUTCSeconds)
+    #[inline]
+    fn set_utc_seconds( &self, args: &[i32] ) -> f64 {
+        // args represents:
+        // [<seconds_value>, <ms_value>]
+        match args.len() {
+            2 => js!( return @{self.as_ref()}.setUTCSeconds(@{args[0]}, @{args[1]}); ).try_into().unwrap(),
+            1 => js!( return @{self.as_ref()}.setUTCSeconds(@{args[0]}); ).try_into().unwrap(),
+            _ => js!( return @{self.as_ref()}.setUTCSeconds(); ).try_into().unwrap()
+
+        }
+    }
+
+    /// DEPRECATED
+    /// fn set_year() -> f64 { }
+
+    ///
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString)
+    #[inline]
+    fn to_date_string( &self ) -> String {
+        js! ( return @{self.as_ref()}.toDateString; ).try_into().unwrap()
```

Comment:
Missing `()` to actually call the `toDateString`.

Reply:
Thanks!